doc/classes/TextureProgressBar.xml

```diff
@@ -42,6 +42,7 @@
 		</member>
 		<member name="radial_initial_angle" type="float" setter="set_radial_initial_angle" getter="get_radial_initial_angle" default="0.0">
 			Starting angle for the fill of [member texture_progress] if [member fill_mode] is [constant FILL_CLOCKWISE], [constant FILL_COUNTER_CLOCKWISE], or [constant FILL_CLOCKWISE_AND_COUNTER_CLOCKWISE]. When the node's [code]value[/code] is equal to its [code]min_value[/code], the texture doesn't show up at all. When the [code]value[/code] increases, the texture fills and tends towards [member radial_fill_degrees].
+			[b]Note:[/b] [member radial_initial_angle] is wrapped in range from to 360 degrees.
```

Comment:
```suggestion
			[b]Note:[/b] [member radial_initial_angle] is wrapped in range from 0 to 360 degrees.
```

Reply:
```suggestion
			[b]Note:[/b] [member radial_initial_angle] is wrapped in range from [code]0[/code] to [code]360[/code] degrees.
```

Reply:
Oh, I accidentally deleted 0... 😆 Thanks!
I think adding the [code] block is a bit much, but I'll leave it as you prefer.

Reply:
Using `code` is preferred to highlight (I missed it)

Reply:
I don't think "wrapped from 0 to 360 degrees" is correct wording, I'd say "wrapped to the [code]0[/code] to [code]360[/code] degree range"

Reply:
Or `wrapped between [code]0[/code] and [code]360[/code] degrees.`